ksqldb-execution/src/main/java/io/confluent/ksql/execution/plan/ForeignKeyTableTableJoin.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright 2019 Confluent Inc.
```

Comment:
nit: update year (and similarly for all the other new files in this PR)

Reply:
What about older files? How do we handle this?